.gitignore

```diff
@@ -58,5 +58,6 @@ out/
 .tmp
 dist/*.map
```

Comment:
Ah, missed this in `publishr` PR. We should remove `dist` from source entirely, because it exists in npm, but will be built from git if dep.